File: Java/src/main/java/banco/State/Rojo.java
package banco.State;

public class Rojo implements IEstado {
    
    public void retirarSaldo(float saldoARet){
    
        if(saldoARet > 1000){
            System.out.println("Puede retirar $1000 como maximo");
            saldoARet = 1000;
        }

        System.out.println("Retira $" + saldoARet);

    }
    
}

File: C#/Banco/State/Rojo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Banco.State
{
    public class Rojo : IEstado
    {

        public void retirarSaldo(float saldoARet) {

            if (saldoARet > 1000)
            {
                Console.WriteLine("Puede retirar $1000 como maximo");
                saldoARet = 1000;
            }

            Console.WriteLine("Retira $" + saldoARet);

        }

    }
}
